pkg/storage/cache.go

```diff
@@ -13,22 +13,24 @@ import (
 
 const (
 	BlobsCache              = "blobs"
+	DBExtensionName         = ".db"
 	dbCacheLockCheckTimeout = 10 * time.Second
 )
 
 type Cache struct {
 	rootDir string
 	db      *bbolt.DB
 	log     zlog.Logger
+	relPath bool // weather or not to use relative paths, should be true for filesystem and false for s3
```

Comment:
"whether"

Reply:
"useRelPath"? I hope I am not reverting some earlier comment

Reply:
np, I'll fix it